### PR TITLE
API user profile changes for UI reg table.

### PR DESCRIPTION
**Pull request by doug-lovett**
Signed-off-by: Doug Lovett <doug@diamante.ca>

*Issue #:* /bcgov/entity#9112

*Description of changes:*
- Update API UserProfile schema to add registrationsTable object type. Value can be any valid JSON.
- UI can use GET/PATCH /ppr/api/v1/user-profile to fetch/update user preferences for the display of the registrations table.  
- 9653 bug fix for financing statement verification report when original secured party, debtor has been updated.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-api/tests/unit/api/test_user_profile.py [hunk 1]
```diff
@@ -100,6 +122,7 @@ def test_update_user_profile(session, client, jwt, desc, staff, include_account,
                       headers=headers,
                       content_type='application/json')
     # check
+    print(rv.json)
```
Comment (kialj876) [inline ppr-api/tests/unit/api/test_user_profile.py]: not sure if you meant to keep this print

**Review by kialj876 — APPROVED** [review]
Looks good, thanks for allowing the generic column names

File: ppr-api/tests/unit/api/test_user_profile.py (hunk 1 above)
Comment (doug-lovett) [reply ppr-api/tests/unit/api/test_user_profile.py]: Thanks for catching this, print statement removed.

**Review by jordiwes — APPROVED** [review]
LGTM

**Pull request merged**